finetune/saver.py

```diff
@@ -258,7 +258,9 @@ def init_fn(scaffold, session, model_portion=None, refresh_base_model=False):
                     saved_var = variables_sv[name]
                 elif name in self.fallback.keys():
                     saved_var = self.fallback[name]
-
+                else:
+                    if 'OptimizeLoss' not in name:
```

Comment:
I think you accidentally left this in. But Maybe we can do some variant of this always? Or behind some flag?

Reply:
Yeah honestly not a bad idea.  Could make a shortlist of weights that should be randomly initialized and then raise a warning if anything else is randomly init-ed.